client/landing/stepper/declarative-flow/ai-site-builder.ts

```diff
@@ -0,0 +1,147 @@
+import { addPlanToCart, addProductsToCart } from '@automattic/onboarding';
+import { useSelect } from '@wordpress/data';
+import { useEffect } from 'react';
+import { useQuery } from 'calypso/landing/stepper/hooks/use-query';
+import { useSiteData } from 'calypso/landing/stepper/hooks/use-site-data';
+import { ONBOARD_STORE } from 'calypso/landing/stepper/stores';
+import wpcom from 'calypso/lib/wp';
+import { useDispatch } from 'calypso/state';
+import { setSelectedSiteId } from 'calypso/state/ui/actions';
+import { stepsWithRequiredLogin } from '../utils/steps-with-required-login';
+import { STEPS } from './internals/steps';
+import { Flow, ProvidedDependencies } from './internals/types';
+import type { OnboardSelect } from '@automattic/data-stores';
+
+export const AI_SITE_BUILDER_FLOW = 'ai-site-builder';
```

Comment:
This const exists in the onboarding package.

Reply:
@kthai15 used it from there